bamboo-storage/src/main/java/com/pushtorefresh/android/bamboostorage/query/DeleteQuery.java

```diff
@@ -16,4 +18,26 @@ public DeleteQuery(@NonNull String table, @Nullable String where, @Nullable Stri
         this.where = where;
         this.whereArgs = whereArgs;
     }
+
+    @Override
+    public boolean equals(@Nullable Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        DeleteQuery that = (DeleteQuery) o;
+
+        if (!table.equals(that.table)) return false;
+        if (where != null ? !where.equals(that.where) : that.where != null) return false;
+        if (!Arrays.equals(whereArgs, that.whereArgs)) return false;
+
+        return true;
+    }
+
+    @Override
+    public int hashCode() {
+        int result = table.hashCode();
+        result = 31 * result + (where != null ? where.hashCode() : 0);
```

Comment:
Yeah, unfortunately no way to use library.
If we will decide to use checkstyle with magic number watchdog, it will be problem =)